app/timer.js

```diff
@@ -25,12 +35,21 @@ define(['jquery'], function($) {
 
   return {
     // Timer functions that initiates and updates the timer.
-    startTimer: function(max_seconds) {
+    startTimer: function(timerid, max_seconds) {
       timer = setInterval(function() { updateTimer(max_seconds); }, 1000);
 
-        $(".timer .max").html(timeToString(max_seconds));
-        $(".timer .current").html("00:00");
-
+      $(timerid+" .max").html(timeToString(max_seconds));
+      $(timerid+" .current").html("00:00");
```

Comment:
The "00:00" is not descriptive enough, if I read it I don't immediately understand what is happening here. Consider storing the "00:00" in a constant and before using it.